cpp/unreal_plugins/UrdfBot/Source/UrdfBot/UrdfLinkComponent.h

```diff
@@ -0,0 +1,29 @@
+//
+// Copyright(c) 2022 Intel. Licensed under the MIT License <http://opensource.org/licenses/MIT>.
+//
+
+#pragma once
+
+#include <vector>
+
+#include <CoreMinimal.h>
+#include <Components/StaticMeshComponent.h>
+
+#include "UrdfLinkComponent.generated.h"
+
+struct UrdfLinkDesc;
+
+class UUrdfJointComponent;
+
+// unreal representation for urdf link
+UCLASS()
+class UUrdfLinkComponent : public UStaticMeshComponent
+{
+    GENERATED_BODY()
+public:
+    // set up link based on link description.
+    void initialize(UrdfLinkDesc* link_desc);
+
+    std::vector<UUrdfLinkComponent*> child_link_components_;
```

Comment:
This state is not necessary, because these pointers will be owned by `UUrdfRobotComponent`.

Reply:
fixed